### ISSUE TITLE
Compiling fails on macOS Sonoma 14.2.1 on a MacBook Pro M3 Max

### ISSUE THREAD

**Issue by johnsonea** [nu774/mp4fpsmod#34]
Compiling fails on macOS Sonoma 14.2.1 on a MacBook Pro M3 Max.  Specifically, in the `mp4v2` subdirectory, it eventually tries to execute the following command:

``` 
/bin/sh ./libtool --tag=CXX --mode=link g++ -std=gnu++11 -Wall -Wformat -g -O2 -fvisibility=hidden -o libutil.la
``` 
which then prints
```
libtool: link: ar cr .libs/libutil.a 
ar: no archive members specified
usage:  ar -d [-TLsv] archive file ...
	ar -m [-TLsv] archive file ...
	ar -m [-abiTLsv] position archive file ...
	ar -p [-TLsv] archive [file ...]
	ar -q [-cTLsv] archive file ...
	ar -r [-cuTLsv] archive file ...
	ar -r [-abciuTLsv] position archive file ...
	ar -t [-TLsv] archive [file ...]
	ar -x [-ouTLsv] archive [file ...]
```

**Comment by nu774** [comment]
Thanks, added github workflow for MacOS and confirmed the issue.
Fixed on the latest commit: e2dd065012f4d2c7e42d4acdefee2ffdc50d3d86.

**Comment by johnsonea** [comment]
Thx for the quick response!  Confirming fixed.
